Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -173,7 +173,9 @@ @implementation WKContentView {
 #if USE(EXTENSIONKIT)
     RetainPtr<id<UIInteraction>> _visibilityPropagationInteractionForWebProcess;
     RetainPtr<id<UIInteraction>> _visibilityPropagationInteractionForGPUProcess;
-#endif
+    RetainPtr<UIView> _visibilityPropagationViewForWebProcess;
+    RetainPtr<UIView> _visibilityPropagationViewForGPUProcess;
```

Comment:
I'm afraid I don't understand why we need to create a separate set of `UIView`s here. Are these meant to represent UI Interactions that should be relayed to the Web and GPU Process? Why are they distinct from the `UIView` that underlies the `WKContentView` object itself?

Reply:
This is not strictly necessary, it was just to make clear that we are creating UIViews in the extension case, and not the specialized sub-classes.

Thanks for reviewing!